tensorflow/python/ops/array_ops.py

```diff
@@ -1931,13 +1931,15 @@ def split(value, num_or_size_splits, axis=0, num=None, name="split"):
   For example:
 
   >>> x = tf.Variable(tf.random.uniform([5, 30], -1, 1))
-
-  Split `x` into 3 tensors along dimension 1
+  >>>
+  >>> # Split `x` into 3 tensors along dimension 1:
+  ...
```

Comment:
The `...` line might not be needed. Same below

Reply:
Thanks @mihaimaruseac Removed.